recipes/pyre/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "pyre" %}
+{% set version = "1.9.5" %}
+{% set config_version = "3.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - folder: pyre
+    url: https://github.com/pyre/pyre/archive/v{{ version }}.tar.gz
+    sha256: 640347883542c06c5bff0f1a02939c0465f0ebcda83fba9097f368cec246c735
+  - folder: config
+    url: https://github.com/aivazis/config/archive/v{{ config_version }}.tar.gz
+    sha256: 2135554b2f3fde6a596e6652ecee19b3ff9839ef260cc2323048d506780afe0a
+
+build:
+  number: 0
+  skip: True  # [win or py2k]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - python {{ python }}
+  host:
+    - python {{ python }}
+  run:
+    - python {{ python }}
```

Comment:
I think both of these can just be `- python`, ie you can drop the `{{ python }}`